templates/download/index.html

```diff
@@ -1,104 +1,111 @@
-    </div>
+{% include "templates/_nav_breadcrumb-v1.html" with section_title="Downloads" page_title="Overview" %}
 {% endblock second_level_nav_items %}
 
 {% block content %}
-<div class="row no-border">
-    <div class="strip-inner-wrapper">
-        <div class="eight-col">
-            <h1>Get Ubuntu</h1>
-            <p>Ubuntu is completely free to download, use and share.</p>
-        </div><!-- / .seven-col -->
-
-        <div class="twelve-col no-margin-bottom ">
-            <div class="box box-highlight no-padding-bottom clearfix">
-                <div class="two-col text-center">
-                    <a href="/download/desktop"><img src="{{ ASSET_SERVER_URL }}4cd0df1c-picto-download-orange.svg" alt="" width="113" height="113" /></a>
-                </div>
-                <div class="ten-col last-col">
-                    <h2><a href="/download/desktop">Ubuntu Desktop&nbsp;&rsaquo;</a></h2>
-                    <p>Download Ubuntu desktop and replace your current operating system whether it&rsquo;s Windows or Mac OS, or, run Ubuntu alongside it.</p>
-                </div>
-                <div class="twelve-col no-margin-bottom">
-                    <p class="ubuntu-upgrade">Do you want to upgrade? <a href="/download/desktop/upgrade">Follow our simple guide&nbsp;&rsaquo;</a></p>
-                </div>
-            </div>
-        </div><!-- /.twelve-col -->
-
-        <div class="twelve-col equal-height no-margin-bottom">
-            <div class="six-col box equal-height">
-                <div class="four-col">
-                    <h2><a href="/download/server">Ubuntu Server&nbsp;&rsaquo;</a></h2>
-                    <p>Whether you want to configure a simple file server or build a fifty thousand-node cloud, you can rely on Ubuntu Server and its five years of guaranteed free upgrades.</p>
-                </div>
-                <div class="two-col last-col align-center equal-height__item equal-height__align-vertically">
-                    <a href="/download/server"><img src="{{ ASSET_SERVER_URL }}0fdf9cfe-picto-server-darkaubergine.svg" alt="" width="113" height="113" /></a>
-                </div>
-            </div>
-            <div class="six-col last-col box equal-height">
-                <div class="four-col">
-                    <h2><a href="/download/cloud">Ubuntu Cloud&nbsp;&rsaquo;</a></h2>
-                    <p>Ubuntu is the reference OS for OpenStack. Canonical&rsquo;s OpenStack Autopilot is a fully automated deployment of an OpenStack cloud on Ubuntu &mdash; just add servers.</p>
-                </div>
-                <div class="two-col last-col align-center equal-height__item equal-height__align-vertically">
-                    <a href="/download/cloud"><img src="{{ ASSET_SERVER_URL }}dfd42685-picto-cloud-darkaubergine.svg" alt="" width="113" height="113" /></a>
-                </div>
-            </div>
+<div class="p-strip">
```

Comment:
This strip needs `is-deep` according to @yaili's pattern spec - https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-overview.png